packages/components/src/color-picker/utils.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+import { each } from 'lodash';
+import tinycolor from 'tinycolor2';
```

Comment:
`tinycolor2` needs to be added to dependencies in `package.json` file:
https://github.com/WordPress/gutenberg/blob/master/packages/components/package.json#L22

Reply:
Fixed at b8b9f37